lib/validation/fileValidationServerSide.tsx

```diff
@@ -5,31 +5,51 @@ import {
   isIndividualFileSizeValid,
 } from "./fileValidationClientSide";
 
-export enum FileValidationResult {
-  VALID,
-  SIZE_IS_TOO_LARGE,
-  INVALID_EXTENSION,
-  INVALID_MIME_TYPE,
-}
+export type FileValidationResult =
```

Comment:
This is my trying to emulate Swift Associated Values Enum feature which is perfect in those situations :)